cobalt/media/sandbox/format_guesstimator.cc

```diff
@@ -95,8 +95,7 @@ std::vector<uint8_t> ReadHeader(const base::FilePath& path) {
   // Size of the input file to be read into memory for checking the validity
   // of ChunkDemuxer::AppendToParseBuffer() calls.
   const int64_t kHeaderSize = 256 * 1024;  // 256kb
-  starboard::ScopedFile file(path.value().c_str(),
-                             kSbFileOpenOnly | kSbFileRead);
+  starboard::ScopedFile file(path.value().c_str(), 0);
```

Comment:
Are we not able to use enums here?

Reply:
SbFileOpen doesn't pass in any flag when seeing kSbFileOpenOnly | kSbFileRead, but we can also specify O_RDONLY in this case, because we assert SB_COMPILE_ASSERT(O_RDONLY == 0, O_RDONLY_must_equal_zero); similarly to SbFileOpen: https://github.com/youtube/cobalt/pull/3243/files#diff-6ae9a2e80c3bbbab34f5554dd45817856978fb17a465bd2bd3103eb643b570b4R436